python/tk_maya/panel_generation.py

```diff
@@ -271,7 +271,7 @@ def build_workspace_control_ui(shotgun_panel_name):
 
     # Retrieve the calling Maya workspace control.
     ptr = MQtUtil.getCurrentParent()
-    workspace_control = wrapInstance(long(ptr), QtWidgets.QWidget)
+    workspace_control = wrapInstance(int(ptr), QtWidgets.QWidget)
```

Comment:
Not sure if this is OK. It doesn't appear to cause issues, and a bit of research seemed to suggest that int and long behaviors are the same in py 2.4 >